src/_guides/language/language-tour.md

```diff
@@ -3693,6 +3696,36 @@ try {
 }
 {% endprettify %}
 
+You can use `await` multiple times in an async function.
+For example, the following code waits three times
+for the results of functions:
+
+<?code-excerpt "misc/lib/language_tour/async.dart (repeated-await)"?>
+{% prettify dart %}
+var entrypoint = await findEntrypoint();
+var exitCode = await runExecutable(entrypoint, args);
+await flushThenExit(exitCode);
+{% endprettify %}
+
+In <code>await <em>expression</em></code>,
+the value of <code><em>expression</em></code> is usually a Future;
+if it isn't, then the value is automatically wrapped in a Future.
```

Comment:
L3711-3712: Is such detail really relevant in a Language Tour?

Reply:
Where else would it go?

Reply:
Given the proposed change in semantics of async functions (which won't block unless they need to), that statement (about non-Future expressions being wrapped) might not even be true anymore. I.e., it is an "implementation detail" of any given compiler or interpreter to wrap or not a non-Future expression.

I'd replace this entire paragraph by the simpler prose along the lines of:

> The expression <code>await <em>expression</em></code> makes execution pause, if necessary, until the value of _expression_ is available

Reply:
I think my wording is still correct. In your wording, I'm not sure about the "if necessary". I think (but should check) that await always pauses.

Reply:
I'm going to leave this as-is (since it's been in the published version for quite a while), but I'll file a bug to remind me to check.

Reply:
Ok, let's drop the "if necessary". That leaves us with:

> The expression <code>await <em>expression</em></code> makes execution pause until the value of _expression_ is available

Reply:
👍 Will be tracked via #521